pipelines/README.md

```diff
@@ -1,45 +1,134 @@
+This page is a good place to start and get an overview of Pipelines approach in general.
+For the details of the exact Pipeline, please, visit the corresponding page:
+* [features_extraction](https://github.com/OML-Team/open-metric-learning/tree/main/pipelines/features_extraction)
+* [postprocessing](https://github.com/OML-Team/open-metric-learning/tree/main/pipelines/postprocessing) (for feature extractor)
+
 ## What are Pipelines?
-Pipelines provide a way to run metric learning experiments via changing only the config file.
-All you need is to prepare the `.csv` table which describes your dataset
-(the format is explained [here](https://open-metric-learning.readthedocs.io/en/latest/oml/data.html)), so the converter
-can be implemented in any programming language.
-
-## How to work with a config?
-We use [Hydra](https://hydra.cc/docs/intro/) as a parser for `.yaml` configs.
-So, you can change whatever you want directly in the config file or override some parameters
-using command line interface:
-```
-python train_cars.py optimizer.args.lr=0.000001 bs_val=128
+
+Pipelines are a predefined collection of scripts/recipes that provide a way to run metric learning
+experiments by changing only the config.
+You need to prepare the `.csv` file which describes your dataset
+(the format is explained [here](https://open-metric-learning.readthedocs.io/en/latest/oml/data.html)).
+
+Pipelines may help you if:
+* You have a dataset which format can be aligned with one required by a pipeline
+* You need reproducibility for your experiments
+* You prefer changing config over diving into the code
+
+They will not work if:
+* You deal with a corner case and the flexibility of an existing Pipeline isn't enough
+
+## Minimal example of a Pipeline
+
+Each Pipeline is built around 3 components:
+* Config file
+* Registry of classes & functions
+* Entrypoint function for a Pipeline + script to run it
```

Comment:
We can write about `.csv` file here.

Reply:
We describe Pipeline here and its components. I don't think it's a good place for mentioning csv here.